src/internal/ajax/ajax.ts

```diff
@@ -12,16 +12,16 @@ export interface AjaxCreationMethod {
    *
    * This is the most configurable option, and the basis for all other AJAX calls in the library.
    *
-   * ### Example
+   * ## Example
```

Comment:
I honestly don't know what the implications are of changing this. @niklas-wortmann, do you have any idea how this might change the docs app output?

Reply:
This changes from `<h3>` to `<h2>`. I did this in couple of files to align them with so many files from `/operators ` folder which all use `##`.